src/deploy-commands.js

```diff
@@ -1,47 +1,71 @@
 const { REST, Routes } = require('discord.js');
-const fs = require('node:fs');
-const path = require('node:path');
 const { DISCORD_SERVER } = require('./config');
 
-module.exports = (client) => {
+const pingCommand = require('./commands/utility/ping');
+const assingMeTaskCommand = require('./commands/utility/assignMeTaskThread');
+const changeStatusCommand = require('./commands/utility/changeStatus');
+const convertTimeCommand = require('./commands/utility/convert-time');
+const myPointsCommand = require('./commands/utility/my-points');
+const prTemplateCommand = require('./commands/utility/pr-template');
+const remindMeCommand = require('./commands/utility/remindme');
+const requestPointCommand = require('./commands/utility/requestPoint');
+const searchCodeCommand = require('./commands/utility/search-code-review');
+const searchMyPointsCommand = require('./commands/utility/search-my-points');
+const sendMessageCommand = require('./commands/utility/send-message');
+const serverCommand = require('./commands/utility/server');
+const userCommand = require('./commands/utility/user');
+const votePointsCommand = require('./commands/utility/vote-points');
+
+const commandList = [
+  pingCommand,
+  assingMeTaskCommand,
+  changeStatusCommand,
+  convertTimeCommand,
+  myPointsCommand,
+  prTemplateCommand,
+  remindMeCommand,
+  requestPointCommand,
+  searchCodeCommand,
+  searchMyPointsCommand,
+  sendMessageCommand,
+  serverCommand,
+  userCommand,
+  votePointsCommand,
+];
```

Comment:
The list is big, and it's weird to have to enter in this file to add or remove commands (since it has some core logic). What about creating this list in another file and only importing the list here? so this file doesn't have to be changed every time we want to add a new command.